### PR TITLE
Add the 'no-restricted-properties' rule

### PR DESCRIPTION
**Pull request by LavrovArtem**

### CODE REVIEW

File: src/client/sandbox/shadow-ui.js [hunk 1]
```diff
@@ -390,37 +390,37 @@ export default class ShadowUI extends SandboxBase {
 
     // Accessors
     getFirstChild (el) {
-        const childNodes = this._filterNodeList(el.childNodes);
+        const filteredEls = this._filterNodeList(el.childNodes);
```
Comment (miherlosev) [inline src/client/sandbox/shadow-ui.js]: `const filteredNodes = ... ` here and above

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Pull request merged**